cli/cli/commands/run/run.go

```diff
@@ -172,6 +176,14 @@ var StarlarkRunCmd = &engine_consuming_kurtosis_command.EngineConsumingKurtosisC
 			Type:    flags.FlagType_String,
 			Default: mainFunctionNameDefaultValue,
 		},
+		{
+			Key: experimentalFeaturesFlagKey,
```

Comment:
Where do I see a list of features? How does this compare to reading features from a config file?


For instruction caching I will have to pass the flag with every run; which can get annoying but if I write it once to config then my life becomes easier. Thoughts?

Reply:
Yes, that's what I wanted to do initially, but then I realized that it's a decent amount more complex. 1. You have to write it in the config file and make sure we support the config file format. Second it's going to drastically complexify Leandro's feature of enclave pooling, because the feature flag can changes between the time the enclave is created and the time it's allocated by the user.
I am not sure this is something we want to keep around, might just be temporary, so I biased toward the easiest. I think anyway those flags are going to be used mostly by Kurtosians, so we can probably eat this amount of pain for a reasonable amount of time. WDYT?